src/components/themed/FilledTextInput.tsx

```diff
@@ -429,7 +449,7 @@ const PlaceholderText = styled(Animated.Text)<{
     useAnimatedStyle(() => {
       return {
         color: interpolatePlaceholderTextColor(focusAnimation, disableAnimation),
-        fontSize: interpolate(shift.value, [0, 1], [fontSizeBase, 0.8 * fontSizeBase])
+        fontSize: interpolate(shift.value, [0, 1], [fontSizeBase, 0.75 * fontSizeBase])
```

Comment:
Ah, could this be the reason for the vertical padding change in the container component? If so, I think keeping the extra padding would give the component more room visually. One of my comments is on the screenshots showing the component to be looking a little crowded when in focus, perhaps we can alleviate this.